### PR TITLE
Improve message formatting

### LINKED ISSUE #67: Improve message formatting
Currently, when I ask a report of my work, the following message is returned.

```
You worked 4.00d in 01 September 2021 - 01 October 2021
 - brand\_identity: 0.13d
 [omissis]
```

This may be improved in several ways, for instance by:

- removing the escape characters as in, e.g., `brand\_identity`,
- Refrasing the first sentence, e.g. "Between 01 September 2021 and 01 October 2021, you worked 4 days (32 hours)",
- Using markdown formatting in the messages.

Steps to reproduce:

1. Open an active chat with the bot
2. Ask "My work this month"

### PR DESCRIPTION
**Pull request by hphan9**
Hi team,
Could you please check my Pull request?
Please let me know if you need anything to be fixed.
fixes #67

### CODE REVIEW

File: src/Common/Resources/Clockify.ClockifyMessageSource.resx [hunk 1]
```diff
@@ -1,157 +1,256 @@
...
+  </data>
+  <data name="RemindStopAnswer" xml:space="preserve">
+    <value>Ok, no more reminders for today, you have my word 🤙</value>
+  </data>
+  <data name="ReportDateRangeExceedOneYear" xml:space="preserve">
+    <value>omg that's a looot of time😰{0}
 Please don't exceed one year window</value>
-    </data>
+  </data>
 </root>
```
Comment (khaelys) [inline src/Common/Resources/Clockify.ClockifyMessageSource.resx:256]: You shouldn't change the entire file, only the required rows

File: src/Clockify/Reports/ReportUtil.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ into groupedTimeEntry
         public static string SummaryForReportEntries(string channel, IEnumerable<ReportEntry> reportEntries)
         {
             var summary = new StringBuilder();
-            var sortedEntries = 
+            var sortedEntries =
```
Comment (khaelys) [inline src/Clockify/Reports/ReportUtil.cs:34]: Do not change the format

File: src/Clockify/Reports/ReportUtil.cs [hunk 3]
```diff
@@ -60,7 +60,7 @@ private static float DurationInDecimal(HydratedTimeEntryDo hydratedTimeEntry)
 
             if (start == null || end == null) return 0.0f;
 
-            return (float) end.Value.Subtract(start.Value).TotalSeconds / 3600;
+            return (float)end.Value.Subtract(start.Value).TotalSeconds / 3600;
```
Comment (khaelys) [inline src/Clockify/Reports/ReportUtil.cs:63]: Do not change the format

File: tests/Bot.Tests/Clockify/Reports/ReportSummaryServiceTest.cs [hunk 4]
```diff
@@ -59,7 +59,7 @@ public async void Summary_UserHasTwoWorkspaces_ReturnsSummaryAggregatedByWorkspa
             string summary = await reportSummaryService.Summary(Channels.Telegram, userProfile, dateRange);
 
             // Assert
-            summary.Should().BeEquivalentTo($"You worked **2.72d** in 01 January 2020 - 01 March 2020" +
+            summary.Should().BeEquivalentTo($"Between 01 January 2020 - 01 March 2020, you worked **2.72d** (21) hours" +
```
Comment (khaelys) [inline tests/Bot.Tests/Clockify/Reports/ReportSummaryServiceTest.cs:62]: You should write the assertion with the new message format, like 2.72 days (21 hours).

File: tests/Bot.Tests/Clockify/Reports/ReportSummaryServiceTest.cs [hunk 5]
```diff
@@ -108,7 +108,7 @@ public async void Summary_UserHasOneWorkspace_DoesNotMentionWorkspaces()
             string summary = await reportSummaryService.Summary(Channels.Telegram, userProfile, dateRange);
 
             // Assert
-            summary.Should().BeEquivalentTo($"You worked **2.28d** in 01 January 2020 - 01 March 2020" +
+            summary.Should().BeEquivalentTo($"Between 01 January 2020 - 01 March 2020, you worked **2.28d** (18) hours" +
```
Comment (khaelys) [inline tests/Bot.Tests/Clockify/Reports/ReportSummaryServiceTest.cs:111]: You should write the assertion with the new message format, like 2.28 days (18 hours).

**Review by khaelys — CHANGES_REQUESTED** [review]

File: tests/Bot.Tests/Clockify/Reports/ReportSummaryServiceTest.cs (hunk 5 above)
Comment (hphan9) [reply tests/Bot.Tests/Clockify/Reports/ReportSummaryServiceTest.cs:111]: Hi @khaelys , thank you for your reply. I will fix that

**Comment by khaelys** [comment]
Hi @hphan9, the only problem is the changes in that XML file. I suggest copying and pasting the original XML file and changing only the line of interest. Another option is to rewrite your commit history, starting clear and force pushing afterward.

**Comment by hphan9** [comment]
Hi @khaelys , Thank you a lot for your support. I just realized I should not edit the .resx file with visual studio, it added buns of things by default. 
I have pulled your late change and I will make a new [Pull request](https://github.com/xtreamsrl/clockify-nlp-bot/pull/117)
Thank you and I am sorry for any inconvenience it might have caused,

**Pull request closed without merging**